var/spack/tests/compilers/cxx/hello.c++

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+
+int main()
+{
+
+	printf ("Hello world from C++\n");
```

Comment:
Should `printf` be used here for a `.c++` file?

Reply:
There are multipe C++ tests. C++ compilers need to be able to handle C-style prints.